tests/components/test-components.php

```diff
@@ -442,14 +442,19 @@ public function test_component_post_list() {
 
 		// Demo templates.
 		$templates = [
-			'before'  => [
+			'before'        => [
 				[ 'name' => 'example/before' ],
 			],
-			'after'   => [
+			'after'         => [
 				[ 'name' => 'example/after' ],
 			],
-			'wrapper' => [ 'name' => 'example/wrapper' ],
-			'item'    => [ 'name' => 'example/item' ],
+			'wrapper'       => [ 'name' => 'example/wrapper' ],
+			'item'          => [ 'name' => 'example/item' ],
+			'interstitials' => [
```

Comment:
I think we should make a separate test specifically for interstitials, which includes multiple positions.